backend/src/cryptography.hpp

```diff
@@ -1,7 +1,11 @@
 #include <vector>
 #include <string>
+#include <cstring>
```

Comment:
**question**: Why do we include this?

Reply:
I think it was an auto import while testing stuff, we can delete it!

Reply:
jk, i think we need it for memcpy